icu4c/source/common/mlbe.cpp

```diff
@@ -128,135 +135,127 @@ int32_t MlBreakEngine::divideUpRange(UText *inText, int32_t rangeStart, int32_t
     return correctedNumBreaks;
 }
 
-void MlBreakEngine::evaluateBreakpoint(UChar32* elementList, int32_t index, int32_t &numBreaks,
-                                         UVector32 &boundary, UErrorCode &status) const {
+int32_t MlBreakEngine::evaluateBreakpoint(const UnicodeString &inString, int32_t *indexList,
+                                          int32_t startIdx, int32_t breakpointIdx,
```

Comment:
why do you need to have both startIdx and breakpointIdx passing in for this function?
You only call this function once and when you call it the value of the 3rd parameter startIdx is always one less than the 4th parameter breakpointIdx. Could re move the int32_t breakpointIdx and later just call
```
boundary.addElement(startIdx+1, status);
```
instead?

Reply:
Done